### ISSUE TITLE
Parameters

### ISSUE THREAD

**Issue by twosg** [skai2/EDAutopilot#1]
The docs say that the SHIP CHARACTERISTICS need to be configured.

But from where can I calculate the values of my specific ship?!
```
; ------------------------------------------------------------------------------
; ---------------------- SECTION TWO: SHIP CHARACTERISTICS ---------------------
; --------------------------You MUST configure these!---------------------------
; ------------------------------------------------------------------------------

; Ship Parameters
Global Pitch360Time = 25700
Global PitchAccelTime = 200
Global PitchAccelAngle = 2
Global PitchAngularSpeed = % 360 / Pitch360Time
Global PitchAccelVal = % PitchAngularSpeed / PitchAccelTime
Global Roll360Time = 5000
Global RollAccelTime = 1300
Global RollAccelAngle = 1
Global RollAngularSpeed = % 360 / Roll360Time
Global RollAccelVal = % RollAngularSpeed / RollAccelTime
Global Yaw360Time = 55300
Global YawAccelTime = 800
Global YawAccelAngle = 1
Global YawAngularSpeed = % 360 / Yaw360Time
Global YawAccelVal = % YawAngularSpeed / YawAccelTime
Global FSDChargeTime = 15000
Global JumpTime = 18000
Global TimeToRefuel = 90000
```

**Comment by skai2** [comment]
You can use the in-game configuration Wizard to actually set the values so there is no need to edit them in the ahk file. However, the only way of getting the numbers is measuring with a stopwatch (ie: pitch360time is how long it takes in ms for your ship to pitch 360 degrees)

**Comment by EnumC** [comment]
Please ignore commit fe00101. I accidentally included "#1" in commit message, and Github assumed that it's relevant to this issue.
